tests/performance_tests/log_reader.py

```diff
@@ -149,4 +151,13 @@ def scoreTransfersPerSecond(data: chainData, numAddlBlocksToDrop=0) -> stats:
     npCBTAEC = np.array(consecBlkTrxsAndEmptyCnt, dtype=np.uint)
 
     # Note: numpy array slicing in use -> [:,0] -> from all elements return index 0
-    return stats(np.min(npCBTAEC[:,0]), np.max(npCBTAEC[:,0]), np.average(npCBTAEC[:,0]), np.std(npCBTAEC[:,0]), np.sum(npCBTAEC[:,1]), len(prunedBlockDataLog))
+    return stats(int(np.min(npCBTAEC[:,0])), int(np.max(npCBTAEC[:,0])), int(np.average(npCBTAEC[:,0])), int(np.std(npCBTAEC[:,0])), int(np.sum(npCBTAEC[:,1])), int(len(prunedBlockDataLog)))
+
+def exportAsJSON(data, args):
+    js = {}
+    js['nodeosVersion'] = os.popen("nodeos --version").read().replace("\n", "")
```

Comment:
Wouldn't this run the nodeos that's in the path? Not the one in the builddir? It seems like `Utils.EosServerName` is what you want assuming the script's working directory is the builddir root (which is how all the tests work, and what I assumed this script was expecting due to the previous `../CMakeLists.txt`)